src/components/task-form.js

```diff
@@ -1,203 +1,140 @@
-const createTaskEditTemplate = () => {
-  return (
-    `<article class="card card--edit card--yellow card--repeat">
-            <form class="card__form" method="get">
-              <div class="card__inner">
-                <div class="card__color-bar">
-                  <svg class="card__color-bar-wave" width="100%" height="10">
-                    <use xlink:href="#wave"></use>
-                  </svg>
-                </div>
+import {COLORS, DAYS, MONTH_NAMES} from "../const.js";
+import {formatTime} from "../utils.js";
+
+// Функция создания разметки цветов, с выбранным цветом (получаем из моков)
+const createColorsMarkup = (colors, currentColor) => {
+  return colors
+    .map((color, index) => {
+      return (
+        `<input
+          type="radio"
+          id="color-${color}-${index}"
+          class="card__color-input card__color-input--${color} visually-hidden"
+          name="color"
+          value="${color}"
+          ${currentColor === color ? `checked` : ``}
+        />
+        <label
+          for="color-${color}--${index}"
```

Comment:
Из-за этого не работает выбор цвета в карточке.
```suggestion
          for="color-${color}-${index}"
```